src/sections/InvolvementSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Box, Center, Wrap } from '@chakra-ui/react';
+import { graphql } from 'gatsby';
+import GetInvolvedCard from 'src/components/GetInvolvedCard';
+import SectionHeader from 'src/components/SectionHeader';
+
+interface Props {
+  data: GatsbyTypes.InvolvmentSectionFragment;
+}
+
+const InvolvmentSection = ({ data }: Props) => {
+  const involvementCards = data?.allContentfulGetInvolvedCard.nodes;
```

Comment:
This needs to filter by locale - see `DonateBanner.tsx` in #47 for an example

Reply:
Do we also want to add the translation for these sections on Contentful or are we doing the translations for all later?